.changeset/shiny-icons-sell.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rocket.chat/i18n': patch
+'@rocket.chat/meteor': patch
+---
+
+Improves clarity of formatting toolbar tooltips.
```

Comment:
Chore PRs don't need changesets. I'm not sure if it is a rule or a suggestion, but either way, such small improvement doesn't need to be in the changelog IMO.